src/main/java/com/google/api/generator/gapic/model/Method.java

```diff
@@ -41,6 +41,9 @@
 
   public abstract boolean isPaged();
 
+  @Nullable
+  public abstract String pageSizeFieldName();
```

Comment:
Should we add a check on `build()` so that folks can't set this without `isPaged`?

Reply:
We can, but doubt it will give practical benefits, but will make the code more verbose and will add too  much manual boilerplate stuff in what is supposed to be boilerplate-autogenerated class (`@AutoValue`). Adding little checks like this moves us from simplistic semantics of `AutoValue` classes, where most of the stuff is generated  in a simplistic and standard way, to not that far away from relying on manually created builders (because this is what we will be doing in `autoValue()` method, which is supposed to be is rarely used feature).

Reply:
I'd prefer that we add this check, so that we can catch slip-ups as early as compile-time (which does occur in practice).

Reply:
Added